tmail-backend/guice/blob-guice/src/test/java/com/linagora/tmail/blob/guice/BlobStoreCacheModulesChooserTest.java

```diff
@@ -10,12 +10,8 @@ public class BlobStoreCacheModulesChooserTest {
     @Test
     void chooseModulesShouldReturnCacheDisabledModuleWhenCacheDisabled() {
         assertThat(BlobStoreCacheModulesChooser.chooseModules(BlobStoreConfiguration.builder()
-<<<<<<< HEAD
```

Comment:
This is a mistake when resolving conflicts, after that creating a new commit to fix the mistake resolve.

IMO, you should redo it again and resolve it correctly.